docker/README.md

```diff
@@ -0,0 +1,95 @@
+# Docker container description for Tendermint applications
```

Comment:
I want to separate building binaries and packaging them for multiple reasons.
1. The build process is a development process requiring a special environment set up. If someone wants to install the baked applications they shouldn't need to install the go language for that. This is what packages are for.
2. I consider a docker container a package type and this way all package building is coded in this repository.
3. This way, the automated package builder (bithead) can build docker packages separated from the build process. Since the build system is undergoing rapid changes (it's still being built), it's easier to change the docker packaging method in the tools repository than in tendermint/basecoin/ethermint repositories.
4. This way the tendermint/basecoin/ethermint repositories can focus on the code and the building method of the binary. If a new package design emerges (say docker++), it doesn't affect these repositories.
5. Most importantly, with the introduction of OpenShift I needed an automatic way of building and uploading docker containers using bithead. Since all code for bithead is in this repo, including the mintnet-kubernetes tool, it made sense to move all docker-related tools in one place.